customTests/azure_ib_write_bw_gdr.nhc

```diff
@@ -22,7 +22,7 @@ function run_ib_bw_gdr(){
    MESSAGE_SIZE=8388608
 
    IB_WRITE_BW=`basename $IB_WRITE_BW_EXE_PATH`
-   IB_WRITE_BW_ARGS="-p $PORTARG -s $MESSAGE_SIZE -x 0 -F --report_gbits  --perform_warm_up"
+   IB_WRITE_BW_ARGS="-p $PORTARG -s $MESSAGE_SIZE -x 0 -F -b --report_gbits  --perform_warm_up"
```

Comment:
Any reason we changed with bi-directional?

Reply:
This was semi related to Issue 46, bidirectional flag increases the perf results. I wanted to get as close to peak perf as possible and bidirectional flag increases the throughput.

Do you see any concerns with using bidirectional traffic on a loop back test?

Reply:
I don't see an issue with using bidirectional with this test. So I am going to continue with it. If an issue arises we can fall back.